velox/exec/tests/CrossJoinTest.cpp

```diff
@@ -246,3 +246,33 @@ TEST_F(CrossJoinTest, parallelism) {
 
   OperatorTestBase::assertQuery(params, "VALUES (30), (30), (30), (30), (30)");
 }
+
+TEST_F(CrossJoinTest, parallelism2) {
```

Comment:
This looks like a copy-paste of the existing test "parallelism". What's the added value of this test? What's the difference from the existing test?

Reply:
Hi Masha,

  In this test there is a hash-agg on the rhs of the cross join as well as one after the projection.

Reply:
That's true, but it looks like this test is not testing any additional functionality then the existing one, does it? It also looks like it didn't uncover any new bugs. If it important to add this query shape, would it be possible to edit the existing test to avoid copy-paste a somewhat complicated comment and making it easier to see the difference between the two plans. E.g. the new code could have a small comment that explain how it is different from the previous query plan.